### ISSUE TITLE
Adding metrics related to ICS29 fees

### ISSUE THREAD

**Issue by ljoss17** [informalsystems/hermes#2945]
## Summary
Retrieve data on ICS29 fees which have been distributed.

> **_Note_** that this work, which depends on Tendermint multi-version support, is currently branched off of the `mikhail/multi-version-tendermint` branch. The last thing to do will be to merge the metrics into `main` once multi-version support has been merged into Hermes.

## Problem Definition

At Hermes v1.1 in order to have an idea of how much fees have been paid to relayers via ICS29, a user would need to query the balances and compute the difference while taking into account the gas paid.

## Proposal

When events with fee distribution information will be emitted, https://github.com/cosmos/ibc-go/issues/2847, Hermes should retrieve and display this information (e.g. adding new metrics).

## Acceptance Criteria

* [x] Hermes displays useful metrics regarding fees distributed via ICS29
* [ ] Merge into `main` once the `mikhail/multi-version-tendermint` branch is merged

____

#### For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate milestone (priority) applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

**Comment by adizere** [comment]
We'll test with https://github.com/cosmos/ibc-go/pull/2975
